arches/app/etl_modules/import_single_csv.py

```diff
@@ -280,12 +280,22 @@ def populate_staging_table(self, loadid, graphid, has_headers, fieldnames, csv_f
                                 config = current_node.config
                                 config["path"] = temp_dir
                                 value = (
-                                    datatype_instance.transform_value_for_tile(source_value, **config) if source_value is not None else None
+                                    datatype_instance.transform_value_for_tile(source_value, **config)
+                                    if source_value is not None and source_value != ""
+                                    else None
```

Comment:
is it possible to change this to:

value = datatype_instance.transform_value_for_tile(source_value, **config) if source_value else None

? Or is there an explicit reason for the `None` and empty string comparison?

Reply:
good point! thanks.